modules/apps/account/account-test/src/testFunctional/tests/2BAccount.testcase

```diff
@@ -683,6 +683,71 @@ definition {
 			assigneeType = "Organizations");
 	}
 
+	@description = "This test asserts that an Account Admin can unassign roles to users."
+	@priority = "5"
+	test AccountAdminCanUnassignRolesToUsers {
+		task ("Given an Account User is given a role by the Account Administrator") {
+			JSONAccountEntryUser.addAccountEntryUser(
+				accountEntryName = "Account Name",
+				userEmailAddress = "user1@liferay.com",
+				userFirstName = "user1",
+				userLastName = "user1",
+				userScreenName = "user1");
+
+			LexiconEntry.gotoEntry(rowEntry = "Account Name");
+
+			Navigator.gotoNavTab(navTab = "Users");
+
+			Accounts.goToEntryMenuItem(
+				menuItem = "Assign Roles",
+				rowEntry = "user1 user1");
+
+			SelectFrame(
+				key_title = "Assign Roles",
+				locator1 = "IFrame#MODAL_ANY");
+
+			Check(
+				key_tableEntry = "Buyer",
+				locator1 = "LexiconTable#TABLE_ENTRY_CHECKBOX");
+
+			SelectFrameTop();
+
+			Button.click(button = "Done");
+
+			AssertTextEquals(
+				key_columnName = "lfr-account-roles-column",
+				key_rowEntry = "user1 user1",
+				locator1 = "ContentRow#ENTRY_CONTENT_ROW_NUMBER_TD",
+				value1 = "Buyer");
+		}
+
+		task ("When the Account Administrator unassigns that role") {
+			Accounts.goToEntryMenuItem(
+				menuItem = "Assign Roles",
+				rowEntry = "user1 user1");
+
+			SelectFrame(
+				key_title = "Assign Roles",
+				locator1 = "IFrame#MODAL_ANY");
+
+			Uncheck(
+				key_tableEntry = "Buyer",
+				locator1 = "LexiconTable#TABLE_ENTRY_CHECKBOX");
+
+			SelectFrameTop();
+
+			Button.click(button = "Done");
+		}
+
+		task ("Then that user should no longer have that role associated with them") {
+			AssertTextNotEquals(
```

Comment:
Could you please double check whether this can be replaced by an exist macro or not? Thanks

Reply:
I wasn't able to find an existing macro that uses the locator _ContentRow#ENTRY_CONTENT_ROW_NUMBER_TD_, but I can create one if you would like in _LexiconTable.macro_ Comment left here: [LRQA-73699](https://issues.liferay.com/browse/LRQA-73699?focusedCommentId=2640840&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-2640840 ) Looks like a good number of our Accounts tests uses that locator with either _AssertTextEquals_ or _AssertTextNotEquals_ at the moment.

Reply:
Thanks for double checking the step @epark18 , It's OK to use the function in testcase if we have no exist macro for it. It can be decided by you whether or not adding the new macro.